### PR TITLE
chore(app-backend): Use v9

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue
Closes: AVO-641
Closes: AVO-649

## Description

Needs: https://github.com/electricitymaps/electricitymaps/pull/5975

Switches the paths to v9 and enables auth for the state path.

It also switches the tokens and env variables to the correct one for v9.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
All hands on deck to review this one to ensure I haven't missed some usage of the old token and that it works as expected with the auth. If you need the new token for local evaluation let me know :)

File: web/Earthfile [hunk 1]
```diff
@@ -36,7 +36,7 @@ build:
   FROM +prepare
   RUN pnpm run create-generated-files
   RUN pnpm version minor
-  RUN --secret SENTRY_AUTH_TOKEN --secret VITE_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN=ELECTRICITYMAP_PUBLIC_TOKEN pnpm run build
+  RUN --secret SENTRY_AUTH_TOKEN --secret VITE_PUBLIC_ELECTRICITYMAP_PUBLIC_TOKEN_V9=ELECTRICITYMAPS_APP_PUBLIC_TOKEN_V9 pnpm run build
```
Comment (tonypls) [inline web/Earthfile:39]: I think we need a backend PR to add this to `.drone-earthly.yml`

**Review by tonypls — APPROVED** [review]
LGTM, approving but let's make the backend PR too and try merge tomorrow

**Comment by VIKTORVAV99** [comment]
> LGTM, approving but let's make the backend PR too and try merge tomorrow

The first backend PR is already merged, that is why this is working ;)

File: web/Earthfile (hunk 1 above)
Comment (tonypls) [reply web/Earthfile:39]: It's in the mono-repo. It's how earthly will get the secrets when it's building the bundle to deploy to prod

Comment (VIKTORVAV99) [reply web/Earthfile:39]: If anyone is following this and for our future self the token did need to be added to the drone config as well. This has now been done.

**Pull request merged**